### ISSUE TITLE
sdk-assistant tries to use zsh

### ISSUE THREAD

**Issue by marmistrz** [sailfishos/sdk-setup#49]
My main shell, set in `/etc/passwd` is zsh. Thus, from `zsh` I run `bash`, from there `sfossdk`. Then 
```
sdk-assistant create $VENDOR-$DEVICE-$PORT_ARCH http://releases.sailfishos.org/sdk/targets-1701-1/Jolla-2.1.0.10-Sailfish_SDK_Target-armv7hl.tar.bz2
```
This fails because of
```
Unpacking target ...
Setting up SB2
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
target_uuidgen: generating a unique id for oneplus-oneplus3-armv7hl target
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
sudo: /usr/bin/zsh: command not found
```

The shell which is set in `/etc/passwd` should have no effect on what's used by `sdk-assistant`, especially that `$SHELL` reads `/bin/bash`

A dirty hack to work this around is to add a symlink `/usr/bin/zsh -> /bin/bash` in the chroot, but it's a dirty hack.

**Comment by martyone** [comment]
PR #50 opened

**Comment by jlehtoranta** [comment]
This issue is fixed in (the upcoming) Sailfish OS >=2.1.3.x. Thanks @marmistrz
